### DISCUSSION TITLE
SSML Support for TTS

### DISCUSSION THREAD

**Discussion by ghost** [deepgram/community#1031] [General help] [answered]
Does deepgram support SSML for TTS?  I read back in March it was something you were looking into?

**Comment by deepgram-community** [comment]
Thanks for asking your question. Please be sure to reply with as much detail as possible so the community can assist you efficiently.
<sub>Consider joining our [Discord community](https://discord.gg/deepgram) for more opportunity to engage with your fellow Deepgram users. You can earn points which can be redeemed for cool stuff by being active in our communities!</sub>

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @g-m-organ, SSML is not on our roadmap at this time. We're seeing that the industry is moving away from SSML, and toward naturally expressive TTS, with accurate pronunciation and emotional inflection. We're planning to release a new version of our TTS, Aura-2, in H1 2025, which will be more expressive and natural-sounding, but it will not have SSML support.

Prior thread for reference: https://github.com/orgs/deepgram/discussions/655

**Reply by ghost** [reply]
Thanks for the reply.  We're having issues with certain pronunciations e.g. our company name BluIP (pronounced Blue IP), any suggestions on how we can handle these issues?  Changing the TTS to pronounce correctly causes an issue with reporting as we would have to map back to the original spelling and were hoping to avoid that.

**Reply by jkroll-deepgram** [reply]
Hi @g-m-organ, unfortunately the only pronunciation control currently is modifying the spelling. You could look into our [extra metadata](https://developers.deepgram.com/docs/extra-metadata) feature to see if you can add extra tracking parameters rather than needing to make use of the TTS text with the modified spelling. We have some pragmatic docs on [TTS pronunciation](https://developers.deepgram.com/docs/text-to-speech-prompting#pronunciation). I did some quick experimentation on our [TTS playground](https://playground.deepgram.com/?endpoint=speak&model=aura-asteria-en), and found that `Blu I P` is pronounced best.
